modules/ROOT/pages/install-patches.adoc

```diff
@@ -0,0 +1,111 @@
+= Perform Scheduled Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following procedure when performing scheduled maintenance for Anypoint Runtime Fabric nodes, such as applying OS patches and security updates. 
+
+== Controller Node Maintenance
+
+The controller nodes are the entry points into your Runtime Fabric cluster but do not run application workloads.
+
+To update controller nodes:
+
+. On your upstream TCP load balancer, remove the controller nodes one at a time from your load balancing pool.
+. Perform the required patching and reboot the node.
+. When updates are complete on each controller node, add the node back to the cluster before removing the next one.
+
+== Worker Node Maintenance
+
+To update worker nodes: 
+
+. On a controller node, run the `gravity shell` command.
+. Verify that the status for all nodes is `READY`:
++
+```
+$ kubectl get nodes
+
+NAME          STATUS   ROLES    AGE   VERSION
+172.31.0.9    Ready    node     1d   v1.x.x
+172.31.0.8    Ready    master   1d   v1.x.x
+```
+. Retrieve the application namespace (environment ID): 
++
+```
+$ kubectl get ns
+
+NAME                                   STATUS   AGE
+default                                Active   1d
+fff5df7b-c49d-48c9-967d-0071412722x4   Active   1d
+kube-node-lease                        Active   1d
+kube-public                            Active   1d
+kube-system                            Active   1d
+monitoring                             Active   1d
+rtf                                    Active   1d
+```
++
+In the command output, the application namespace is represented by a long string of characters. In the previous example, the application namespace
+is `fff5df7b-c49d-48c9-967d-0071412722x4`.
++
+More than one application namespace value is displayed if the RTF cluster is associated with more than one environment.
+
+. Verify that the pods and nodes are still running in your namespace:
++
+```
+$ kubectl -n <namespace> get pod -owide
+```
++
+In the command output, verify that `Running` is displayed in the `READY` column, as shown in the following example:
++
+```
+NAME                                         READY   STATUS    RESTARTS   AGE     IP             NODE          NOMINATED NODE   READINESS GATES
+my-app-name-1-84f95cb7c9-glktk     2/2     Running   0          9m27s   10.244.88.29   172.31.0.9   <none>           <none>
+```
++
+If you have multiple environments, repeat the remaining steps in each environment. 
```

Comment:
What does this mean?
If you have one environment, stop the task here?
And if you do not, drain the node, verify, patch and reboot, and reenable scheduling?
You don't need to perform any of the above for a single environment?

We might need three subtasks here:
controller node
worker node, one environment
worker node, multiple environments